### ISSUE TITLE
Bug: The pages aren't dynamic

### ISSUE THREAD

**Issue by feralheart** [ProfessionalWiki/Network#61]
In this [article](https://professional.wiki/en/news/mediawiki-page-network) you says that the graph is dynamically expanding, but both in 1.4 and 2.0 this feature is missing

**Comment by krabina** [comment]
I agree with @feralheart: I cannot get the feature shown in sec 22 to work. Dynamically expanding the node does not seem to work.

**Comment by JeroenDeDauw** [comment]
Using the latest version, I can expand nodes. Not exactly smoothly though.

Example: https://maps.extension.wiki/wiki/Network_example Click "KML export"

Issues:
* You need to click the area the arrow points to. The text is sometimes displayed on a weird location, outside the clickable area. Looks like a regression caused by styling changes.
* It does not work for all pages, due to the Display Title bug added in 2.0: https://github.com/ProfessionalWiki/Network/issues/58

**Comment by krabina** [comment]
Even after your hint, I cannot manage. Two times it suddenly happened, but I fail to reproduce this. I can double-click which leads me to the page, but expanding seems to be impossible.

**Comment by JeroenDeDauw** [comment]
You need to click here. If you hover on the correct spot, it also shows the page name in a little popover thing.

![image](https://user-images.githubusercontent.com/146040/187546656-dc789608-0e02-4aee-b406-8e96f7c109ea.png)

**Comment by krabina** [comment]
Yes, I see the hover, but when I click, nothing happens. I might be old, but I am not so old that I am not able to click on something. This is clearly a bug.

**Comment by JeroenDeDauw** [comment]
You don't click. You HOLD

**Comment by krabina** [comment]
Indeed! How shoud anybody know this. So it is not a bug, but an enhancement, to provide some kind of info about how this works.
